internal/state/dataplane/configuration.go

```diff
@@ -91,11 +91,11 @@ func (r *MatchRule) GetMatch() v1beta1.HTTPRouteMatch {
 	return r.Source.Spec.Rules[r.RuleIdx].Matches[r.MatchIdx]
 }
 
-// buildConfiguration builds the Configuration from the graph.
+// BuildConfiguration builds the Configuration from the Graph.
 // FIXME(pleshakov) For now we only handle paths with prefix matches. Handle exact and regex matches
-func buildConfiguration(
+func BuildConfiguration(
 	ctx context.Context,
-	graph *graph,
+	graph *graph.Graph,
```

Comment:
Can we use a different variable name? `graph` shadows the package name.

Reply:
yeah (I also couldn't think of a better name for package or the type)